src/docfx/build/toc/ServicePageGenerator.cs

```diff
@@ -48,7 +48,7 @@ public void GenerateServicePageFromTopLevelTOC(TocNode node, List<FilePath> resu
             var referenceTOCFullPath = Path.GetFullPath(Path.Combine(_docsetPath, referenceTOCRelativeDir));
             var nodeHrefFullPath = Path.GetFullPath(Path.Combine(referenceTOCFullPath, node.Href.Value ?? ""));
 
-            if (string.IsNullOrEmpty(node.Href.Value) || !File.Exists(nodeHrefFullPath))
```

Comment:
This looks hack to me. Is it possible to determine by checking whether there is a file of that name already existing in the repo?

Reply:
Fount the way to get fullpath when start with ~